sentences/da/_common.yaml

```
@@ -46,10 +106,176 @@ lists:
       - "fahrenheit"
       - in: "f"
         out: "fahrenheit"
+
+  # binary_sensor
+  bs_battery_states:
+    values:
+      - in: "lav"
+        out: "on"
+      - in: "normal"
+        out: "off"
+  bs_battery_charging_states:
+    values:
+      - in: "lader"
+        out: "on"
+      - in: "lader ikke"
+        out: "off"
+  bs_carbon_monoxide_states:
+    values:
+      - in: "(detekteret|påvist|tilstedeværende)"
```

I don't think "påvist" and "tilstedeværende" make sense, semantically, for this.

```suggestion
      - in: "(detekteret|fundet|registreret)"
```